improver/feels_like_temperature.py

```diff
@@ -212,53 +161,77 @@ def calculate_feels_like_temperature(temperature, wind_speed,
     in order to blend between the wind chill and the apparent temperature.
 
     Args:
-      temperature (iris.cube.Cube):
-        Cube of air temperatures
+        temperature (numpy.ndarray)
+        apparent_temperature (numpy.ndarray)
+        wind_chill (numpy.ndarray)
```

Comment:
This doc-string could specify the required units, like the calculate apparent temperature method does.

Reply:
The units are mentioned at the top of the docstring: `... inputs in degrees Celsius ...`.  Is that sufficient?

Reply:
I have no strong feelings about this, so you can leave it if you wish.